dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/TracingInterceptor.kt

```diff
@@ -130,8 +143,9 @@ internal constructor(
     // region Internal
 
     private fun isRequestTraceable(request: Request): Boolean {
-        val host = request.url().host()
-        return host.matches(hostRegex)
+        val url = request.url()
+        return CoreFeature.firstPartyHostDetector.isFirstPartyUrl(url) ||
```

Comment:
I would rather inject the `firstPartyHostDetector` in the constructor. Not a big fan of having this static reference inside this function here.